higher/optim.py

```diff
@@ -31,7 +31,7 @@
 _GradClosureType = _typing.Callable[[_torch.Tensor], _torch.Tensor]
 _OverrideType = _typing.Dict[str, _typing.List[_typing.Any]]
 _GradCallbackType = _typing.Callable[[_typing.List[_torch.Tensor]],
-                                      _typing.List[_torch.Tensor]]
+                                     _typing.List[_torch.Tensor]]
```

Comment:
I liked the spacing the old way. Any reason this was changed?
```suggestion
                                      _typing.List[_torch.Tensor]]
```

Reply:
I've noticed these changes too before making the PR. I followed the Google coding style as instructed in the contribution guidelines and used their linter to lint my code. Since the linter and the existing code style conflicted, I wasn't sure what to go with so I just followed the guidelines. I'm happy to change them back if you'd like.

Reply:
Don't worry. It's most likely that I misconfigured my linter.